### PR TITLE
Emporous rebrand

### PR DESCRIPTION
**Pull request by ned1313**
Updates to the website to support the rebranding from UOR to Emporous. The following changes have been made:

* Changed all references of UOR Framework to Emporous Framework
* Changed all references of uor-framework to emporous
* Changed uor client references to emporous
* Changed client.uor-framework.io to client.emporous.io
* Updated logo name from uor to emporous

External changes that will be required:

- [x] Update website to use emporous.io
- [ ] Update client name to emporous
- [ ] Update API to use emporous
- [x] Rename website repo to emporous.io

Outstanding items:

* Getting started is not 100% working as documented due to client changes that are in flux

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [uor-framework] [UOR Framework]
+   Copyright [emporous-framework] [Emporous Framework]
```
Comment (usrbinkat) [inline LICENSE:189]: We can remove "framework" here

File: blog/2022-08-21-welcome/index.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 slug: welcome
 title: Welcome
 authors: [afflom, jpower432, sabre1041, usrbinkat]
-tags: [uor, hello]
+tags: [uor, hello, emporous]
 ---
 
-The UOR Framework community welcomes your participation! Questions, feedback, and contributions are the life blood of any new project and we hope to hear from you!
+The Emporous Framework community welcomes your participation! Questions, feedback, and contributions are the life blood of any new project and we hope to hear from you!
```
Comment (usrbinkat) [inline blog/2022-08-21-welcome/index.md:8]: We can remove framework here

File: docs/intro.md [hunk 3]
```diff
@@ -4,15 +4,15 @@ sidebar_position: 1
...
 # FAQ
 
-Let's cover some basics of UOR.
+Let's cover some basics of Emporous.
 
-## What is UOR?
+## What is Emporous?
 
-Universal Object Reference (UOR) is a framework used to generalize multiple streams of content management into a single content management mechanism and strategy.
+Emporous is a universal object reference framework used to generalize multiple streams of content management into a single content management mechanism and strategy.
```
Comment (usrbinkat) [inline docs/intro.md:11]: We can remove framework here

File: docs/intro.md [hunk 4]
```diff
@@ -27,40 +27,37 @@ Didn’t we say anything? A few examples of how UOR can be used are:
...
+This might seem arbitrary at first glance, but it is a fundamental concept in human cognition. We describe a table to another person via its attributes i.e. Dark wood, 18x2in rectangular legs, round top... If we’ve been precise enough in our description, another person would be able to pick that table out of a showroom of tables. Emporous takes this concept and applies it to everything. We can then train AI models on a uniformly formatted internet containing contextually linked data. 
 
 ### What’s this about embedded algorithms?
 
-UOR provides a simple way to embed application logic into collections of content. These embedded algorithms are called Universal Runtime Objects (UROs). There are several benefits from this approach such as:  
+Emporous provides a simple way to embed application logic into collections of content. These embedded algorithms are called Universal Runtime Objects (UROs). There are several benefits from this approach such as:  
 
 - Version locked compatibility between application and content.  
-- Enables UOR to be used as a serverless framework  
+- Enables Emporous to be used as a serverless framework
```
Comment (usrbinkat) [inline docs/intro.md:54]: We can remove framework here

File: docusaurus.config.js [hunk 5]
```diff
@@ -136,7 +136,7 @@ const config = {
             ],
           },
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} UOR Framework`,
+        copyright: `Copyright © ${new Date().getFullYear()} Emporous Framework`,
```
Comment (usrbinkat) [inline docusaurus.config.js:139]: We can remove framework here

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # How to Contribute
 
-Of course we welcome contributions to the UOR Framework Documentation!
+Of course we welcome contributions to the Emporous Framework Documentation!
```
Comment (usrbinkat) [inline CONTRIBUTING.md:3]: We can remove framework here

**Review by usrbinkat — CHANGES_REQUESTED** [review]
Good start, let's remove instances of the word `framework` where it's no-longer required due to renaming.

**Comment by usrbinkat** [comment]
Added a PR to your fork https://github.com/ned1313/universalreference.io/pull/1

**Comment by ned1313** [comment]
PR merged from @usrbinkat and references to framework removed.

**Review by usrbinkat — APPROVED** [review]
Thank you for your contribution!

File: blog/2022-08-21-welcome/index.md [hunk 7]
```diff
@@ -2,9 +2,9 @@
 slug: welcome
 title: Welcome
 authors: [afflom, jpower432, sabre1041, usrbinkat]
-tags: [uor, hello]
+tags: [uor, hello, emporous]
```
Comment (sabre1041) [inline blog/2022-08-21-welcome/index.md:5]: Remove reference to `uor`

**Review by sabre1041 — COMMENTED** [review]
Found one more reference to `uor` that we would most likely want to remove. 

Otherwise this looks great!

File: blog/2022-08-21-welcome/index.md (hunk 7 above)
Comment (ned1313) [reply blog/2022-08-21-welcome/index.md]: Wasn't sure if we wanted to keep it for historical reasons. I have removed it.

**Review by sabre1041 — APPROVED** [review]
LGTM

File: LICENSE [hunk 8]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [uor-framework] [UOR Framework]
+   Copyright [emporous] [Emporous]
```
Comment (jpower432) [inline LICENSE:189]:
```suggestion
   Copyright 2022 Emporous Authors.
```

**Review by jpower432 — APPROVED** [review]

**Pull request merged**